### PR TITLE
[Intl] Add missing entries to intl upgrade log

### PR DESCRIPTION
**Pull request by malteschlueter**
| Q             | A
| ------------- | ---
| Branch?       | 5.3
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

This is a follow up of #41361.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
All these classes are internal AFAIK. If confirmed, there is no need for these lines.

**Comment by malteschlueter** [comment]
> All these classes are internal AFAIK. If confirmed, there is no need for these lines.

Good to know, thank you^^

Only the deleted Exceptions are not `@internal` but they just used in the deleted so it doesn't matter or?

File: UPGRADE-5.3.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ HttpKernel
  * Deprecate `HttpKernelInterface::MASTER_REQUEST` and add `HttpKernelInterface::MAIN_REQUEST` as replacement
  * Deprecate `KernelEvent::isMasterRequest()` and add `isMainRequest()` as replacement
 
+Intl
+----------
+
+ * Deprecate `DateFormatter` namespace, use symfony/polyfill-intl-icu ^1.21 instead
```
Comment (stof) [inline UPGRADE-5.3.md:69]: These are internal classes. What should be mentioned are the polyfill classes for PHP class names

**Comment by malteschlueter** [comment]
Okay, I will close this PR. Thank you @nicolas-grekas and @stof

**Comment by stof** [comment]
There is still a need to mention that the polyfills bundled in the component are deprecated, and that you shuld install the polyfill package if you rely on them.

**Comment by stof** [comment]
for the exception, the `@internal` tag should be added IMO. not having it was a mistake.

**Pull request closed without merging**